.github/labeler.yaml

```diff
@@ -0,0 +1,36 @@
+area/docs:
+- "docs/**/*"
+- "**/*.md"
+
+area/config:
+- "config/*"
+- "config/**/*"
```

Comment:
Suggestion: The label `area/config` sounds  little confusing to me because here we are only keeping CRDs, RBAC manifests. I guess maybe this is not required at all or we could label it `area/apis`. @anusha94 @jamiemonserrate

Reply:
Hmmm makes sense. I'm okay to not having config label.

Reply:
I'm fine removing the `area/config` label as it won't be updated frequently

As per my understanding `area/apis` should only consist of pure contracts and nothing else. We already have a dedicated label for `apis`.